ksp_plugin/plugin.cpp

```diff
@@ -719,8 +694,10 @@ not_null<std::unique_ptr<Plugin>> Plugin::ReadFromMessage(
                  std::move(celestials),
                  std::move(bubble),
                  std::move(ephemeris),
-                 prolongation_integrator,
-                 prediction_integrator,
+                 /*prolongation_integrator=*/DormandElMikkawyPrince1986RKN434FM<
+                     Position<Barycentric>>(),
+                 /*prediction_integrator=*/DormandElMikkawyPrince1986RKN434FM<
+                     Position<Barycentric>>(),
```

Comment:
`prolongation_parameters` and `prediction_parameters`, serialized, defaulted only once (in the constructor).